charts/neolink/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+appVersion: 0.3.0
+description: Neolink - RTSP bridge to Reolink IP cameras
+name: neolink
+version: 1.0.0
+keywords:
+  - reolink
+  - rtsp
+home: https://github.com/k8s-at-home/charts/tree/master/charts/neolink
+sources:
+  - https://github.com/thirtythreeforty/neolink
+maintainers:
+  - name: mr-onion-2
```

Comment:
Did you want to add email here as well?

Reply:
No thanks, my other PR was to _remove_ my email :)
I saw some other charts without email so guessed that would be OK

Reply:
Oh I thought your last pr was too add your email. nevermind